plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -181,6 +181,9 @@ builtinMemoryModels =
     , paramValueContains = Id $ ModelTwoArgumentsConstantCost 32
     , paramValueData = Id $ ModelOneArgumentConstantCost 32
     , paramUnValueData = Id $ ModelOneArgumentConstantCost 32
+    , paramInsertCoin = Id $ ModelFourArgumentsConstantCost 1
+    , paramUnionValue = Id $ ModelTwoArgumentsConstantCost 1
+    , paramScaleValue = Id $ ModelTwoArgumentsConstantCost 1
```

Comment:
Put something here!
- insertCoin is linear in its value argument
- unionValue is linear in (arg1 + arg2)
- scaleValue is linear in its value argument